scene/main/scene_tree.cpp

```diff
@@ -1451,6 +1451,9 @@ SceneTree::SceneTree() {
 	bool snap_2d_vertices = GLOBAL_DEF("rendering/2d/snap/snap_2d_vertices_to_pixel", false);
 	root->set_snap_2d_vertices_to_pixel(snap_2d_vertices);
 
+	const bool measure_render_time = GLOBAL_DEF("rendering/measure_render_time", false);
+	RS::get_singleton()->viewport_set_measure_render_time(root->get_viewport_rid(), measure_render_time);
```

Comment:
Won't think break if the view_fps label is toggled? 

E.g.
https://github.com/godotengine/godot/blob/49a196277f86977d08cc10b6bb878427ae54a964/editor/plugins/node_3d_editor_plugin.cpp#L2800

I have a feeling that ``viewport_get_measured_render_time_gpu()`` should just throw a warning when ``viewport->measure_render_time`` is false.

Reply:
The project setting shouldn't have an effect in the editor itself, but it's true that you can technically query the value from an editor plugin (albeit unlikely).